rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -447,3 +753,109 @@ DECLARE(ppu_module_manager::cellCamera)("cellCamera", []()
 	REG_FUNC(cellCamera, cellCameraSetNotifyEventQueue2);
 	REG_FUNC(cellCamera, cellCameraRemoveNotifyEventQueue2);
 });
+
+void camera_thread::on_task()
+{
+	while (fxm::check<camera_thread>() && !Emu.IsStopped())
+	{
+		std::chrono::steady_clock::time_point frame_start = std::chrono::steady_clock::now();
+
+		if (Emu.IsPaused())
+		{
+			std::this_thread::sleep_for(1ms); // hack
+			continue;
+		}
+
+		for (auto const& notify_data_entry : notify_data_map)
```

Comment:
you will need a lock around this, otherwise notify_data_map could change during the loop